api/src/services/email/email.service.ts

```diff
@@ -13,6 +16,14 @@ const emailService = new EmailService({
   },
 });
 
+const emailAlternativeService = new EmailService({
```

Comment:
You can change `EmailService` to have ability to override `from` argument. So we dont have to create instances for each `from` variant
```js
 return sendgrid.send({
      to,
      from: from || this.from,  
      templateId,
      dynamicTemplateData,
    });    
```